server/constants/vars.js

```diff
@@ -20,6 +20,19 @@ export const DEFAULT_CHART_FILTERS = {
   ],
 }
 
+export const INITIAL_CHART_FILTERS = {
```

Comment:
why do we need a new object for this? Can we update `DEFAULT_CHART_FILTERS` instead? Also, do we need a similar change on the backend?

Reply:
hmm, could use your input here.
We have this condition.
https://github.com/AMP-SCZ/dpdash/blob/e89cce01288d72520f4a6094b17ccfe0b9fda650/server/controllers/chartsController/index.js#L108-L110

This changes everything because we want all data, but the initial query only has a few select filters

Reply:
ah yeah I see. I recommend updating `DEFAULT_CHART_FILTERS` and creating a new object, `ALL_FILTERS_ACTIVE`, which can be used in the chartsController making 

```js
const allFiltersSelected = deepEqual(filters, DEFAULT_CHART_FILTERS) 
```

become

```js
const allFiltersSelected = deepEqual(filters, ALL_FILTERS_ACTIVE) 
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/269/commits/e3f4bbfaa6339b60c0870ef73a8b0fe1572f57c1

Reply:
I think we want `INITIAL_CHART_FILTERS` to be `DEFAULT_CHART_FILTERS` though.

Reply:
like we can still have default chart filters, and this ticket is changing the default chart filters for the cohort group.

Reply:
👍 
https://github.com/AMP-SCZ/dpdash/pull/269/commits/a8ee77eae54592f938129b35b7247d54f2583a6e